src/pages/Home/index.js

```diff
@@ -0,0 +1,534 @@
+import React from "react";
+import {
+  Typography,
+  CssBaseline,
+  Container,
+  Button,
+  Card,
+  Stack,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useStyles from "./styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import CardMedia from "@mui/material/CardMedia";
+import CardContent from "@mui/material/CardContent";
+
+function Home() {
+  const isActive = useMediaQuery("(max-width:600px)");
+  const isActiveIpad = useMediaQuery("(max-width:1300px)");
+  const classes = useStyles();
+  return (
+    <>
+      <CssBaseline />
+      <main>
+        <div className={classes.container}>
+          <Container maxWidth="md">
+            <Typography
+              variant="h4"
+              align={isActive ? "left" : "center"}
+              className={!isActive && classes.heading}
+              color="textPrimary"
+              gutterBottom
+            >
+              Let income not be a barrier to your career dreams
+            </Typography>
+            <Typography
+              variant="h6"
+              align={isActive ? "left" : "center"}
+              gutterBottom
+            >
+              With Meraki, begin your programming journey for free today
+            </Typography>
+            <div>
+              <Grid
+                sx={{ mt: 3 }}
+                container
+                spacing={2}
+                justifyContent="center"
+              >
+                <Grid item>
+                  <Button
+                    className={
+                      isActive ? classes.responsiveBtn : classes.LearningBtn
+                    }
+                    variant="contained"
+                    color="primary"
+                  >
+                    Start Learning
+                  </Button>
+                </Grid>
+                <Grid item>
+                  <Button
+                    className={
+                      isActive ? classes.responsiveBtn : classes.LearningBtn
+                    }
+                    variant="outlined"
+                    color="primary"
+                  >
+                    <img
+                      className={classes.playstoreImg}
+                      src={require("./assets/playstore.svg")}
+                    />
+                    <span className={classes.downloadBtn}>Download Meraki</span>
+                  </Button>
+                </Grid>
+              </Grid>
+            </div>
+          </Container>
+
+          {/* ********************** Section 2 ******************* */}
+
+          <Container sx={{ mt: 4, display: { xs: "none", md: "flex" } }}>
+            <Grid container>
```

Comment:
This should create the gap without adding unnecessary HTML via an empty `Grid` item.
```suggestion
            <Grid container justifyContent="space-between">
```